fuzz.go

```diff
@@ -0,0 +1,31 @@
+// +build gofuzz
+
+package sdp
+
+// Fuzz implements a randomized fuzz test of the sdp
+// parser using go-fuzz.
+//
+// To run the fuzzer, first download go-fuzz:
+// `go get github.com/dvyukov/go-fuzz/...`
+//
+// Then build the testing package:
+// `go-fuzz-build`
+//
+// And run the fuzzer on the corpus:
+// `go-fuzz`
+func Fuzz(data []byte) int {
+	// Check that unmarshalling any byte slice does not panic.
+	var sd SessionDescription
+	if err := sd.Unmarshal(data); err != nil {
+		return 0
+	}
+	// Check that we can marshal anything we unmarshalled.
+	_, err := sd.Marshal()
+	if err != nil {
+		panic("failed to marshal")
```

Comment:
https://travis-ci.org/github/pion/sdp/jobs/690765726#L270
```
$ bash .github/lint-disallowed-functions-in-library.sh
/home/travis/gopath/src/github.com/pion/sdp/.github/../fuzz.go:		panic("failed to marshal")
panic( may only be used in example code
```
please add `// nolint`

Reply:
Done. If we add many more fuzz functions we'll probably want to make an exception for any file guarded by `// +build gofuzz`.